io/src/lzf_image_io.cpp

```diff
@@ -127,9 +127,9 @@ pcl::io::LZFImageWriter::compress (const char* input,
   {
     // Copy the header first
     const char header[] = "PCLZF";
-    memcpy (&output[0],  &header[0], 5);
-    memcpy (&output[5],  &width, sizeof (std::uint32_t));
-    memcpy (&output[9],  &height, sizeof (std::uint32_t));
+    std::copy_n(header, 5, output);
+    std::copy_n(&width, sizeof(std::uint32_t), output + 5);
+    std::copy_n(&height, sizeof(std::uint32_t), output + 9);
```

Comment:
Based on another change, please revert any modification to memcpy which is copying from one type to another.  This file has a few places